cloud-aws/src/main/resources/templates/aws-cf-dbstack.ftl

```diff
@@ -87,7 +97,42 @@
   },
 
   "Resources" : {
-
+  <#if hasSecurityGroup>
+  <#else>
```

Comment:
<#if !hasSecurityGroup> should work, but switching to that isn't a show-stopper for me.

Reply:
Ah, great, just found another part of the ftl manual.